core-blocks/embed/index.js

```diff
@@ -28,6 +29,24 @@ import './editor.scss';
 // These embeds do not work in sandboxes
 const HOSTS_NO_PREVIEWS = [ 'facebook.com' ];
 
+// Caches the embed API calls, so if blocks get transformed, or deleted and added again, we don't spam the API.
+const wpEmbedAPI = memoize( ( url ) => wp.apiRequest( { path: `/oembed/1.0/proxy?${ stringify( { url } ) }` } ) );
+
+const matchesPatterns = ( url, patterns ) => {
+	return patterns.some( ( pattern ) => {
```

Comment:
Noting that this line can error:

1. Paste `https://wordpress.org/gutenberg/handbook` into a new paragraph block

```
Uncaught TypeError: Cannot read property 'some' of undefined
```

Simple solution may just be to add a default parameter value for `patterns`:

```js
const matchesPatterns = ( url, patterns = [] ) => {
```